### ISSUE TITLE
Add a way to mute a topic

### ISSUE THREAD

**Issue by bgilbert** [zulip/zulip-android#10]
There doesn't seem to be any UI for muting a topic.

**Comment by bgilbert** [comment]
This could go in the context menu available by long-pressing a message.

**Comment by Swati4star** [comment]
@kunall17 This enhancement is there in the app. Shouldn't we close this one now?

**Comment by kunall17** [comment]
@Swati4star Nope, its not there the code is there, I had implemented this working but there was no sync with the server at the time so it was temporarily removed If you can work on the server for the API it would be great! 

@timabbott Do we have endpoint API for muting topic/stream/private message?

**Comment by timabbott** [comment]
Hmm, we don't have an API endpoint for muting right now, but it'd be easy to add one; we'd basically just need to move `set_muted_topics` from a legacy non-API URL to an API URL.

**Comment by timabbott** [comment]
(I can help with this if someone is interested in using that feature)
